src/modules/mavlink/mavlink_receiver.h

```diff
@@ -407,12 +406,9 @@ class MavlinkReceiver : public ModuleParams
 	hrt_abstime _heartbeat_component_uart_bridge{0};
 
 	DEFINE_PARAMETERS(
-		(ParamFloat<px4::params::BAT_CRIT_THR>)     _param_bat_crit_thr,
-		(ParamFloat<px4::params::BAT_EMERGEN_THR>)  _param_bat_emergen_thr,
-		(ParamFloat<px4::params::BAT_LOW_THR>)      _param_bat_low_thr,
-		(ParamFloat<px4::params::MAV_HITL_LAT>) _param_hil_home_lat,
```

Comment:
This  looks a lot like re-doing the patch :

commit 9b9a2793eb59e34b25188b908b35e545f4dadea6
Author: haitomatic <hai.to@unikie.com>
Date:   Fri Jan 19 12:36:39 2024 +0000

    mavlink: enable support for gzsim HITL msg reception

If that is so, could we first revert the original patch and then put this on?

Reply:
sorry i tried to squash https://github.com/tiiuae/px4-firmware/commit/9b9a2793eb59e34b25188b908b35e545f4dadea6 with https://github.com/tiiuae/px4-firmware/pull/941/commits/78618976a0d05d83e0fd05f225a20a190c2ce0ec but it fails :disappointed:

Reply:
Can I do this latter after merging to https://github.com/tiiuae/px4-firmware/blob/v1.15.3_rebase ? maybe that would be easier..

Reply:
commit https://github.com/tiiuae/px4-firmware/commit/9b9a2793eb59e34b25188b908b35e545f4dadea6 is now reverted. We can remove it latter in the v1.15.3 rebase branch.